libs/utils/src/votes.ts

```diff
@@ -84,3 +88,28 @@ export const buildVoteFromCvr = ({
 
   return vote
 }
+
+/**
+ * Gets all the vote options a voter can make for a given yes/no contest.
+ * @returns ContestVoteOption[] ex. ['yes', 'no']
+ */
+export function getContestVoteOptionsForYesNoContest(
+  contest: YesNoContest
+): readonly YesNoVoteID[] {
+  assert(contest.type === 'yesno')
```

Comment:
```suggestion
  assert.equal(contest.type, 'yesno')
```
Though I wonder why this is here given the type of `contest` is already `YesNoContest`.